pipenv/core.py

```diff
@@ -2922,7 +2923,8 @@ def do_check(
     if not quiet and not project.s.is_quiet():
         click.echo(
             click.style(
-                decode_for_output("Checking installed package safety..."), bold=True
+                decode_for_output("Checking installed packages for vulnerabilities..."),
```

Comment:
Please don't use decode for output. It's not needed any more.

Reply:
Done! 👍